docs/framework-app_dynamics_agent.md

```diff
@@ -17,14 +19,14 @@ When binding AppDynamics using a user-provided service, it must have name or tag
 
 | Name | Description
 | ---- | -----------
```

Comment:
Pretty much none of these changes are correct.  None of `account-access-key`, `account-name`, `application-name`, `port`, `ssl-enabled`, `tier-name` are required.